package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Fastify Mysql connection plugin",
   "main": "index.js",
   "scripts": {
-    "test": "standard && tap test/*.test.js"
+    "test": "standard && tap test/*.test.js",
+    "docker": "docker run -d -p 3306:3306 -e MYSQL_ALLOW_EMPTY_PASSWORD=yes mysql"
```

Comment:
Can you call the script `mysql`?
You can also add `--rm` as we did [here](https://github.com/fastify/fastify-redis/blob/master/package.json#L8) :)